src/components/Text.js

```diff
@@ -4,10 +4,31 @@ import React from 'react';
 // Homepage About Text
 export const WelcomeText = () => {
   return (
+    // <div>
+    //   <div>
+    //     <p>in the pace of this world, we sometimes struggle to keep up especially in times (Covid, economy, BLM) understanding this is our natural instinct to keep up w the pace,
+    //     we wanted to provide a tool to be able to capture a certain moment(takeaways, influences, thoughts )
+    //     </p>
+    //     <p>But how? In order to prevent us from hopping on the next quick thing,
+    //     for future reference not only captures but is also a reminder of the current circumstances in order to strengthen us and one another.
+    //     Following a time of reflection we want to give the opportunity to give encouragement and thanks to the people who’ve helped us along the way.
+    //     </p>
+    //   </div>
+    // </div>
     <div>
       <div>
-        <p> A tool to capture the present, remember the past, and refresh your outlook on the future.
-        </p>
+        {/* <font size="7"> */}
+          {/* <p><i>Reflect</i>, Remember, and <i>Repeat</i>. */}
+          {/* </p> */}
+        {/* </font> */}
+        {/* <font size="3">
+          <p>A tool to <i>capture</i> the present, <i>remember</i> the past, and to <i>refresh</i> your outlook on the future.
+          </p>
+        </font> */}
+        <font size="7">
+          <p> A tool to <i>capture</i> the present, <i>remember</i> the past, and <i>refresh</i> your outlook on the future.
+          </p>
+        </font>
```

Comment:
Delete if unused. If you're just leaving it as filler for now, just put it in (uncommented out) or some other filler like 'coming soon'